config/packages/test/twig.yaml

```diff
@@ -1,2 +0,0 @@
-twig:
```

Comment:
I think all twig related files are removed by accident please make sure twig is listed as a dependency

Reply:
Twig is installed and 'composer require twig' makes no changes